seeders/08_users.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  up: queryInterface => {
+    const faker = require('faker');
+    const range = require('lodash/range');
+    const md5 = require('md5');
+
+    const arr = range(1, 100).map((value, index) => {
+      const createdBefore = parseInt(Math.random() * 1000);
+      return {
+        id: index,
+        first_name: faker.name.firstName(),
+        last_name: faker.name.lastName(),
+        email: faker.internet.email(),
+        password: md5(faker.internet.password()),
```

Comment:
instead of faking the password please use a string so that we know how to login using the seeded users. 

so something like 

```
password: md5('node-password'),
```